src/app/pages/datasets/components/dataset-management/dataset-management.component.ts

```diff
@@ -52,6 +68,12 @@ export class DatasetsManagementComponent implements OnInit {
       });
   }
 
+  ngAfterViewInit(): void {
+    if (window.innerWidth < 1280) {
```

Comment:
Magic number needs to be extracted somewhere. If this breakpoint matches what Material uses, I think you can import and use `Breakpoints/BREAKPOINTS` constants.

Reply:
I found that Breakpoints also contains a BreakpointsObserver I can subscribe to. I switched to using that instead of hostlistener